client/db/bolt/upgrades.go

```diff
@@ -143,3 +146,36 @@ func v1Upgrade(dbtx *bbolt.Tx) error {
 		return nil
 	})
 }
+
+// v2Upgrade adds a MaxFeeRate field to the OrderMetaData. The upgrade sets the
+// MaxFeeRate field for all historical orders to the max uint64. This avoids any
+// chance of rejecting a pre-existing active match.
+func v2Upgrade(dbtx *bbolt.Tx) error {
+	const oldVersion = 1
+	const newVersion = 2
+
+	dbVersion, err := fetchDBVersion(dbtx)
+	if err != nil {
+		return fmt.Errorf("error fetching database version: %w", err)
+	}
+
+	if dbVersion != oldVersion {
+		return fmt.Errorf("v2Upgrade inappropriately called")
+	}
+
+	// For each order, set a maxfeerate of max uint64.
+	maxFeeB := uint64Bytes(^uint64(0))
+
+	master := dbtx.Bucket(ordersBucket)
+	if master == nil {
+		return fmt.Errorf("failed to open orders bucket")
+	}
+
```

Comment:
Need to update the stored version to 2, no?  `setDBVersion(dbtx, newVersion)` and a check in `verifyV2Upgrade`